src/content.js

```diff
@@ -53,6 +53,7 @@ async function getPageMetadata(contentDir) {
             ...attributes,
             _md: body,
             _dir,
+            _slug: _dir[_dir.length - 1],
```

Comment:
The last part of the URL path, added here for convenience in other code using page metadata.